docs/users-documentation.html

```diff
@@ -380,45 +389,65 @@
                  
              <li>
                     <span class="strong">Usage</span>
-                    ./da-cli.sh difference [--json] leftID rightID
+                    ./da-cli.py difference [--json] leftID rightID
                     <ul>
                         <li>--json Output unparsed response from Dependency Analyzer.</li>
                         <li>leftID ID of first product</li>
                         <li>rightID ID of second product</li>
                     </ul>
                 </li>
                </ul>
+        
+        
+        
+        
+        
+        <li>
+        <span class="strong">Administration tools</span>
+                <ul>
+                <li>
+                    These tools add, remove or change data. These are administrative tools, so you need to be authenticated to use them.
+                    You can authenticate with your credentials when you run admin tool. You are asked to enter your login. If no is entered, 
+                    your system user name is used. After you are asked to enter password. In case credentials are wrong, CLI tool exits with failure message with no data changed.
```

Comment:
"If no is entered, your system user name is used."
I would rephrase this sentence. Maybe saying that the system user name is default?